docs/process-services/directives/process-audit.directive.md

```diff
@@ -1,43 +0,0 @@
----
-Title: Process Audit Directive
-Added: v2.0.0
-Status: Active
-Last reviewed: 2019-01-14
----
-
-# [Process Audit Directive](../../../lib/process-services/src/lib/process-list/components/process-audit/process-audit.directive.ts "Defined in process-audit.directive.ts")
-
-Fetches the Process Audit information in PDF or JSON format.
-
-![adf-process-audit-directive](../../docassets/images/adf-process-audit-directive.png)
```

Comment:
I think you can remove that image from assets because it's unused now

Reply:
thanks, removed